addon/components/fixtable-grid.js

```diff
@@ -159,15 +163,24 @@ export default Ember.Component.extend({
       var filters = this.get('filterToApply') || {};
       var sortedFilteredContent = sortedContent.filter(function(row) {
         return Object.keys(filters).every(function(columnKey) {
-          if (!filters[columnKey]) { return true; } // no filter
-          var cellData = (((row.get) ? row.get(columnKey) : row[columnKey]) || '').toLowerCase();
-          var filterValue = filters[columnKey].toLowerCase();
+          var filterDef = columnsByKey[columnKey].filter;
+          var filterFunction = filterDef.function;
```

Comment:
I'm glad you said something about this. I am trying to be DRY and I didn't expected this to work. When it did, I thought there has to be _something_ that will break.

I'll change it to `filterFunction`.